ruby/sorting-words-and-numbers/main.rb

```diff
@@ -1,16 +1,19 @@
 # frozen_string_literal: true
+SEPARATER = ' '
 
-loop do
-  sorted = [[], []]
-  line = gets
-  next if line.nil?
+while line = gets.chomp
+  classified_words = {
+    numerical_digits: [],
+    others: [],
```

Comment:
### Why `others` ?

`strings` is a bad name, because the value stored in numerical_digits is also a string type value.

`words` is also a bad name, because it is words that will be handled in the looping process, and the value stored in numerical_digits is also a word.
Also, if a specification is added in the future that says "we want to classify uppercase letters as well", adding `uppercase_words` will duplicate words and is not MECE.

In consideration of future additions to the specification, I decide that `others` would be a better name.